spec.html

```diff
@@ -30616,7 +30617,7 @@ <h1>Properties of the Symbol Constructor</h1>
 
       <emu-clause id="sec-symbol.asynciterator">
         <h1>Symbol.asyncIterator</h1>
-        <p>The initial value of `Symbol.asyncIterator` is the well-known symbol @@asyncIterator (<emu-xref href="#table-well-known-symbols"></emu-xref>).</p>
+        <p>The initial value of `Symbol.asyncIterator` is the well known symbol %Symbol.asyncIterator% (<emu-xref href="#table-well-known-symbols"></emu-xref>).</p>
```

Comment:
Reinstate hyphen, just for consistency:
```suggestion
        <p>The initial value of `Symbol.asyncIterator` is the well-known symbol %Symbol.asyncIterator% (<emu-xref href="#table-well-known-symbols"></emu-xref>).</p>
```